tests/analyzer/test_data_diagnosis.py

```diff
@@ -332,9 +333,12 @@ def test_mutli_rules(self):
                         'metrics': ['kernel-launch/wall_overhead']
                     },
                     'rule3': {
-                        'categories': 'CNN',
-                        'function': 'multi_rules',
-                        'criteria': 'lambda label:True if label["rule1"]+label["rule2"]>=2 else False'
+                        'categories':
+                        'CNN',
+                        'function':
+                        'multi_rules',
+                        'criteria':
+                        'lambda label:True if label["rule1"]["violated"]+label["rule2"]["violated"]>=2 else False'
```

Comment:
Why use this format?

Reply:
I tried Yifan's suggestions, but nothing changed